### PR TITLE
Feature/vertex ai predictions

### PR DESCRIPTION
**Pull request by wardVD**

### CODE REVIEW

**Comment by remoyson** [comment]
@wardVD Good idea to include it in this repo! 
Would it be possible to
-  update the main readme with  domain ML in production (just like the other ones)
- update the MLIP domain readme with current content (also just like the other ones)

File: mlip/2021_06_09_vertex_ai_predictions/deploy_model_vertexai.ipynb [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+   "outputs": [],
+   "source": [
+    "! pip3 install google-cloud-storage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parameter settings"
```
Comment (remoyson) [inline mlip/2021_06_09_vertex_ai_predictions/deploy_model_vertexai.ipynb:32]: shouldn't we mention here some prerequisites that we except to have a GCP project and a model already in a bucket?

File: mlip/2021_06_09_vertex_ai_predictions/deploy_model_vertexai.ipynb [hunk 2]
```diff
@@ -0,0 +1,502 @@
+{
```
Comment (remoyson) [inline mlip/2021_06_09_vertex_ai_predictions/deploy_model_vertexai.ipynb:1]: Would include some intro with some context (f.e. only for tensorflow in this notebook)

File: mlip/2021_06_09_vertex_ai_predictions/deploy_model_vertexai.ipynb [hunk 3]
```diff
@@ -0,0 +1,502 @@
...
+    "API_ENDPOINT = \"{}-aiplatform.googleapis.com\".format(REGION)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def upload_model_sample(\n",
```
Comment (remoyson) [inline mlip/2021_06_09_vertex_ai_predictions/deploy_model_vertexai.ipynb:194]: Think this sample function is directly from a ai platform sample provided by Google. Shouldn't we mention that somewhere?

**Comment by wardVD** [comment]
@remoyson Updated PR to include your comments.

**Pull request merged**